### ISSUE TITLE
[Bug]: Sparkline <path> element does not have associated role (aria-allowed-attr)

### ISSUE THREAD

**Issue by siwmao** [microsoft/fluentui#29936]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (20) x64 13th Gen Intel(R) Core(TM) i7-13800H
    Memory: 35.07 GB / 63.66 GB
  Browsers:
    Edge: Chromium (119.0.2151.93)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/sparkline

### Bug Description

We are leveraging `@fluentui/react-charting`'s `Sparkline` and `GaugeChart` components for a few of our react blades. As part of release readiness, we run the Accessibility Insights for Web's FastPass automated checker. We noticed that in the Sparkline component, the [aria-allowed-attr ](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr/) check is failing. the <path> elements contain the aria-label but no associated role. This can  be reproduced by going to the provided examples and running FastPass on the `Sparkline `example:

![sparkline_fastpass](https://github.com/microsoft/fluentui/assets/137838190/e9ec68eb-c1a2-4647-853d-183903bd83f8)

## Actual Behavior
The FastPass automated check fail due to the <path> element containing an aria-label attribute with no valid role attribute. 
## Expected Behavior
The associated <path> element should have a corresponding role attribute. The FastPass automated checks should pass.

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by AtishayMsft** [comment]
@siwmao Thanks for reporting. We are aware of this issue and tracking to fix it in December.
